### PR TITLE
Update German translations

### PR DESCRIPTION
**Pull request by der-lyse**

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -676,9 +676,8 @@ msgid "Already on first feed."
 msgstr "Bereits beim ersten Feed."
 
 #: src/feedlistformaction.cpp:409
-#, fuzzy
 msgid "Do you really want to mark all feeds as read (y:Yes n:No)? "
-msgstr "Wollen Sie wirklich `%s' überschreiben (y:Ja n:Nein)? "
+msgstr "Möchten Sie wirklich alle Feeds als gelesen markieren (j:Ja n:Nein)? "
```
Comment (dennisschagt) [inline po/de.po:680]: Related to using `j:Ja`, there  seems to be one place where still `y:Ja` is shown which might be good to switch to the `j`-key as well.
https://github.com/newsboat/newsboat/blob/a087abf63e4df66a9d06133d7440b40dcf018d00/po/de.po#L790

The used keys are specified here (showing that `j` and `n` are used):
https://github.com/newsboat/newsboat/blob/a087abf63e4df66a9d06133d7440b40dcf018d00/po/de.po#L686

Comment (der-lyse) [reply po/de.po:680]: You're absolutely right, I force-pushed a fix (and also changed to German typographic quotes when updating that line).

Comment (Minoru) [reply po/de.po:680]: @der-lyse, looks like you updated just the quotes; the "y" is still there.

There's also something funny going on with CI: most jobs went fine, but a few of them failed to find your branch. No idea what's going on, let's hope it fixes itself when you push again. :crossed_fingers:

Comment (der-lyse) [reply po/de.po:680]: Oh dear, looks like it's way too late for me to do some proper work… :-( My apologies for this terrible mess! And thanks for spotting.

Earlier I accidentally force-pushed my branch to the official newsboat repo and not to my own fork. Shortly after I noticed and corrected that. Maybe that's what confused CI. Anyway, I now tried again with the "y" → "j" and have to go to bed. See you tomorrow!

Comment (Minoru) [reply po/de.po:680]: Thanks! I believe this is resolved, but since this is @dennisschagt's review, let's wait for him to confirm.

Comment (dennisschagt) [reply po/de.po:680]: Looks good, thanks for fixing this

**Review by dennisschagt — APPROVED** [review]
LGTM

**Comment by Minoru** [comment]
Thanks, to both of you!

**Pull request merged**